frontend/src/app/shared/components/table-pagination/table-pagination.component.ts

```diff
@@ -65,6 +65,9 @@ export class TablePaginationComponent extends UntilDestroyedMixin implements OnI
     label_next: this.I18n.t('js.pagination.pages.next'),
     per_page: this.I18n.t('js.label_per_page'),
     no_other_page: this.I18n.t('js.pagination.no_other_page'),
+    pagination_controls: this.I18n.t('js.pagination.pagination_controls'),
+    pageNumberText: (num:number) => { return this.I18n.t('js.pagination.pages.page_number', { number: num }); },
+    show_per_page: (num:number) => { return this.I18n.t('js.pagination.pages.show_per_page', { number: num }); },
```

Comment:
nitpick - can you make the function naming consistent? either kebab or snake-case.

you can also use the shorter, expression-body syntax here:
```suggestion
    pageNumberText: (num:number) => this.I18n.t('js.pagination.pages.page_number', { number: num }),
    show_per_page: (num:number) => this.I18n.t('js.pagination.pages.show_per_page', { number: num }),
```